h2o-core/src/main/java/water/api/ImportSQLTableHandler.java

```diff
@@ -20,7 +20,8 @@ public JobV3 importSQLTable(int version, ImportSQLTableV99 importSqlTable) {
     if (importSqlTable.fetch_mode == null) {
       sqlFetchMode = SqlFetchMode.DISTRIBUTED;
     } else {
-      sqlFetchMode = EnumUtils.valueOfIgnoreCase(SqlFetchMode.class, importSqlTable.fetch_mode);
+      sqlFetchMode = EnumUtils.valueOfIgnoreCase(SqlFetchMode.class, importSqlTable.fetch_mode)
```

Comment:
I would do something similar to what you are doing in getMethodFromUserInput() i.e. include the invalid value in the exception message

Reply:
Done. Changed to:

```java
.orElseThrow(() -> new IllegalArgumentException("Unrecognized SQL Fetch mode: " + importSqlTable.fetch_mode));
```